src/js/trove/load-lib.js

```diff
@@ -333,7 +333,23 @@
           // NOTE(joe): This is the place to add checkAll
           var currentChecker = otherRuntime.getField(checker, "make-check-context").app(otherRuntime.makeString(main), checkAll);
           otherRuntime.setParam("current-checker", currentChecker);
-        }
+        },
+        "builtin://table": function(table) {
```

Comment:
Crud.  I'm noticing that this set of postLoadHooks is diverging from handalone.js (and probably cpo-standalone) too.  Already, handalone mentions "throwInvalidTableColumn" and "toArray" in the ffi section, and this doesn't; this doesn't include builtin://data-source or builtin://reactors.  We need to make sure these three loaders stay in sync.